### DISCUSSION TITLE
Is there a way to ignore non-HTTPS requests?

### DISCUSSION THREAD

**Discussion by dstd** [mitmproxy/mitmproxy#5636] [Q&A]
I got the point that it's impossible to ignore non-HTTPS requests by domain due to the engine limitations.

But in my case most of such requests are the garbage like connectivity check pings (eg: http://connectivitycheck.gstatic.com/generate_204), so I can just ignore them all to keep my flows list clean.

Is it possible?

**Comment by mhils** [comment]
I think the pragmatic approach is to set a view filter here so that they are just not shown. Or you write a custom addon that deletes them in the `response` hook.
